go/test/endtoend/onlineddl/scheduler/onlineddl_scheduler_test.go

```diff
@@ -555,6 +558,8 @@ func testScheduler(t *testing.T) {
 			onlineddl.CheckMigrationStatus(t, &vtParams, shards, t1uuid, schema.OnlineDDLStatusRunning)
 			onlineddl.CheckMigrationStatus(t, &vtParams, shards, t2uuid, schema.OnlineDDLStatusQueued, schema.OnlineDDLStatusReady)
 		})
+		onlineddl.CheckThrottledApps(t, &vtParams, throttlerapp.OnlineDDLName, false)
```

Comment:
Adding a "wait-for", because `onlineddl.ThrottleAllMigrations` is now an async flow.